modules/op-creating-pipeline-tasks.adoc

```diff
@@ -31,6 +31,11 @@ update-deployment                 48 seconds ago
 
 . Use the `tkn clustertasks list` command to list the Operator-installed additional ClusterTasks, for example --`buildah` and `s2i-python-3`:
 +
+[NOTE]
+====
+You must use a privileged Pod container to run `buildah` ClusterTask as it requires a privileged security context. To learn more about Security Context Constraints (SCC) for Pods, see Additional resources section.
```

Comment:
Some missing articles and suggested word change:

"You must use a privileged Pod container to run **the** `buildah` ClusterTask **because** it requires a privileged security context."

"See **the** Additional resources section."

Reply:
@lamek - I have incorporated your review comments. Request you to review, and if you are ok with the changes please merge it to 4.4 and 4.5. Thank you so much :)